src/tiny.ts

```diff
@@ -427,7 +434,7 @@ export class TinyPg {
 
             const result = await new Promise<Pg.QueryResult>((resolve, reject) => {
                // The type definition does not know .callback is a function.
-               (<any>query).callback = (err: any, res: any) => (err ? reject(err) : resolve(res))
+               ;(<any>query).callback = (err: any, res: any) => (err ? reject(err) : resolve(res))
```

Comment:
prettier?

Reply:
maybe to avoid the prettier stuff we can extract this single line to a function.

Reply:
@joeandaverde how about this https://github.com/joeandaverde/tinypg/pull/18/commits/6896cd4cf8aa0aad9cbb15d047052c13e8d1706f ?